src/java.se/share/data/jdwp/jdwp.spec

```diff
@@ -44,7 +44,7 @@ JDWP "Java(tm) Debug Wire Protocol"
     (Command ClassesBySignature=2
         "Returns reference types for all the classes loaded by the target VM "
         "which match the given signature. "
-        "Multple reference types will be returned if two or more class "
+        "Multiple reference types will be returned if two or more class "
```

Comment:
This file's name scares me. The fact that there's no "serviceability" label on this PR adds to this feeling. I would ask around if I were you.

Reply:
I'll revert all changes in this file. I don't want it to hold up the entire PR. We can revisit it later, with serviceability.